src/containers/pages/AssigmentMapWrapper/index.tsx

```diff
@@ -32,18 +49,93 @@ export interface AssignmentMapWrapperProps {
   currentChildren: TreeNode[];
   serviceClass: typeof OpenSRPService;
   fetchJurisdictionsActionCreator: typeof fetchJurisdictions;
+  fetchUpdatedCurrentParentActionCreator: typeof fetchUpdatedCurrentParent;
   getJurisdictionsFeatures: FeatureCollection;
+  autoSelectNodesActionCreator: ActionCreator<AutoSelectNodesAction>;
+  selectNodeCreator: ActionCreator<SelectNodeAction>;
+  deselectNodeCreator: ActionCreator<DeselectNodeAction>;
+  currentParentNode: TreeNode | undefined;
 }
 
 const defaultProps = {
+  autoSelectNodesActionCreator: autoSelectNodes,
   currentChildren: [],
   currentParentId: undefined,
+  currentParentNode: undefined,
+  deselectNodeCreator: deselectNode,
   fetchJurisdictionsActionCreator: fetchJurisdictions,
+  fetchUpdatedCurrentParentActionCreator: fetchUpdatedCurrentParent,
   getJurisdictionsFeatures: undefined,
   rootJurisdictionId: '',
+  selectNodeCreator: selectNode,
   serviceClass: OpenSRPService,
 };
 
+/**
+ * Handler to get current parent id from clicked jurisdiction on map
+ * @param props - AssignmentMapWrapper component props
+ */
+
+export const onJurisdictionClick = (props: any, setMapParent: any) => {
```

Comment:
Can we move this function to a utils or helpers module? Something like `../AssigmentMapWrapper/{utils/helpers}.tsx `